src/cmakeTools.ts

```diff
@@ -1470,22 +1452,22 @@ export class CMakeTools implements vscode.Disposable, api.CMakeToolsAPI {
         return drv ? this.tasksBuildCommandDrv(drv) : null;
     }
 
-    private m_promise_build: Promise<number> = Promise.resolve(0);
+    private mPromiseBuild: Promise<number> = Promise.resolve(0);
```

Comment:
remove the `m`. It looks like C++ naming convention left over.  Might be better to call it `activeBuild` if we want to remove the type name 'promise' from the variable (generally desirable).